docs/reference/config.rst

```diff
@@ -1,39 +1,37 @@
 Configuration
 =============
 
-Beets has an extensive configuration system that lets you customize nearly
-every aspect of its operation. To configure beets, you create a file called
-``config.yaml``. The location of the file depend on your platform (type ``beet
-config -p`` to see the path on your system):
-
-* On Unix-like OSes, write ``~/.config/beets/config.yaml``.
-* On Windows, use ``%APPDATA%\beets\config.yaml``. This is usually in a
-  directory like ``C:\Users\You\AppData\Roaming``.
-* On OS X, you can use either the Unix location or ``~/Library/Application
+Beets has an extensive configuration system which lets you customize nearly
+every aspect of beets. To configure beets, a configuration file called
+``config.yaml`` must be created. The location of ``config.yaml`` depends on your operating system (OS).
+
+* On a Unix OS, use ``~/.config/beets/config.yaml``.
```

Comment:
I know it sounds silly, but “Unix-like OSes” is more accurate. Linux isn’t exactly Unix (even if it is a POSIX), and  Cygwin is even less of a Unix, but this advice applies there too.

Reply:
I see! Thanks for clarifying